.ddev/commands/web/stan

```diff
@@ -16,8 +17,15 @@ then
   exit
 fi
 
+# Level setting.
```

Comment:
We could replace this with `getopts`, but in my testing, everything worked fine without it.